.github/workflows/docs-deploy.yml

```
@@ -51,6 +51,10 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pnpm-store-
 
+      # https://github.com/postmanlabs/postman-code-generators/pull/806
+      - name: Remove yarn from PATH
+        run: sudo rm -f $(which yarn)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:12: Quote this to prevent word splitting

(shellcheck)